packages/babel-types/src/definitions/es2015.js

```diff
@@ -249,7 +249,7 @@ defineType("ClassDeclaration", {
 });
 
 defineType("ExportAllDeclaration", {
-  visitor: ["source"],
+  visitor: ["source", "exported"],
```

Comment:
What is this needed for?

Reply:
We won't remove identifierName from the location if we don't visit it.

Reply:
Oh ok, I was confused because it isn't in `fields`. Since it is ESTree-specific and not part of the Babel AST, maybe we should monkey patch it like we do it [here](https://github.com/babel/babel/blob/ffb42488ba1a9d05a41d028cd72ae8c77dfe2071/eslint/babel-eslint-parser/src/convert/convertAST.js#L86)?

Reply:
Good catch!